CMakeLists.txt

```diff
@@ -13,21 +13,38 @@ option(BUILD_FORTRAN "Build the Fortran Mex libraries" OFF)
 
 # Add cmake directory to CMake's path
 if(NOT CMAKE_MODULE_PATH)
-  set(CMAKE_MODULE_PATH)
+    set(CMAKE_MODULE_PATH)
 endif()
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
 # Look for packages early so we can exit if they're not found
 find_package(MPI REQUIRED)
+include(herbert_Version)
 include(herbert_FindMatlab)
 include(herbert_AddMex)
 if(${BUILD_TESTS})
-  include(herbert_FindGTest)
-  enable_testing()
+    include(herbert_FindGTest)
+    enable_testing()
 endif()
 
 # Set the destination of our build artifacts
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
 
 add_subdirectory("_LowLevelCode")
+add_subdirectory("admin")
+
+# =============================================================================
+# Install commands
+# =============================================================================
+include(herbert_CPackConfig)
+install(
+    DIRECTORY "herbert_core/"
+    DESTINATION "Herbert"
+    USE_SOURCE_PERMISSIONS
+    PATTERN "*.m~" EXCLUDE
```

Comment:
are there C++ build files still kicking around that will get picked up (`.o`, `.obj`)?

Reply:
Not in `herbert_core` no. We assume an out-of-source build. I have neglected to put the temporary Matlab file extension for Windows in though - I'll add that